src/project.py

```diff
@@ -114,6 +113,22 @@ def on_container(func):
                 )(func)
 
 
+
+@MyProject.label
+def rdfed(job):
+    return job.isfile("rdf.txt")
+
+
+def on_pflow(func):
+    import socket
+    hostname = socket.gethostname()
+    if hostname == "fry.boisestate.edu":
+        return flow.directives(
+            executable='$HOME/miniconda3/envs/planckton-flow/bin/python')(func)
+    elif hostname.endswith(".bridges2.psc.edu"):
+        return flow.directives(
+            executable='$HOME/.conda/envs/planckton-flow/bin/python')(func)
```

Comment:
```suggestion
    import sys                                            
                                                          
    pypath = sys.executable                                                                     
    return flow.directives(executable=f'{pypath}')(func)  
    
```
Here is a solution that instead of checking the hostname--instead passes the python path from when the job is submitted (planckton-flow env python) to the job. I've tested it on Fry and will test it on Bridges.

Reply:
Just tested this out and it works great, thanks Jenny!